### ISSUE TITLE
put setupDB as an optional/configuration flag

### ISSUE THREAD

**Issue by SaMnCo** [AmitKrVarman/PolicyValidationAPI#2]
I noticed your app does a DB setup when it starts. This is not optimal in the Kubernetes world. To manage these, we create an additional step called "init" which runs a separate container. See https://kubernetes.io/docs/concepts/workloads/pods/init-containers/ for reference. 

So that means that I need an app that will "just" do the setup, and another that will expect the DB to be filled. 

2 options as this app is very small: 

1. You add for me a tag I can add at startup (go run app --setup=true or whatever)
2. You create a separate app that is dedicated to loading the DB

I hope it is clear let me know otherwise.

**Comment by AmitKrVarman** [comment]
Yes it would make sense to break the API into two small units - one for setup and another to ascess data.

I will do this in next version for now I have added flag to skip setup.

Please pass command line agrument "true" to execute setup (go run *.go true)
